game/constants.py

```diff
@@ -10,4 +10,11 @@
 
 TILE_SIZE = 32 * TILE_SPRITE_SCALING
 
-FONT_NAME = "game/assets/fonts/NewTegomin-Regular.ttf"
+INGAME_UI_FONT = "game/assets/fonts/NewTegomin-Regular.ttf"
+MENU_FONT = "game/assets/fonts/NewTegomin-Regular.ttf"
```

Comment:
this font kinda looks really bad in the menu, ngl. it feels too piratey for a main menu